ouroboros-consensus-test/ouroboros-consensus-test.cabal

```diff
@@ -313,3 +313,35 @@ test-suite test-infra
                        -Wredundant-constraints
                        -Wmissing-export-lists
                        -fno-ignore-asserts
+
+-- test-examples is just used to type-check tutorial code:
+                          
+test-suite test-examples
```

Comment:
Is this test-suite still needed? Seems like the tutorials have been moved to another package 🤔